cfbs.json

```diff
@@ -834,7 +834,20 @@
       "by": "https://github.com/cfengine",
       "version": "3.21.2",
       "commit": "f495603285f9bd90d5d36df4fec4870aeee751e8",
-      "steps": ["run ./prepare.sh -y", "copy ./ ./"]
+      "steps": [
+        "run EXPLICIT_VERSION=3.21.2 EXPLICIT_RELEASE=1 ./prepare.sh -y",
+        "delete cfe_internal/core/watchdog/README.md",
```

Comment:
Do we still want all these various delete steps?

Reply:
Yes, we want build steps to match exactly between versions.json and cfbs.json.

In newer versions of `prepare.sh` (AKA next LTS, I guess) they are not needed.

Reply:
Yes, if we want the masterfiles release tarball to match the cfbs build result.

Without it, I see those files that were deleted are only from the cfbs built version.

```
$ cf-remote --version 3.21.2 download masterfiles
Available releases: master, 3.22.0, 3.21.x, 3.21.2, 3.21.1, 3.21.0, 3.18.x, 3.18.5, 3.18.4, 3.18.3, 3.18.2, 3.18.1, 3.18.0
Using 3.21.2 LTS (default):
Downloading package: '/home/nickanderson/.cfengine/cf-remote/packages/cfengine-masterfiles-3.21.2-1.pkg.tar.gz'
$ tar zxvf /home/nickanderson/.cfengine/cf-remote/packages/cfengine-masterfiles-3.21.2-1.pkg.tar.gz
$ mv masterfiles masterfiles-3.21.2-pkg
$ diff -ur out/masterfiles masterfiles-3.21.2-pkg
Only in out/masterfiles: cfbs.json
Only in out/masterfiles/cfe_internal/core/watchdog: README.md
Only in out/masterfiles/cfe_internal/enterprise/ha: ha_info.json
Only in out/masterfiles: .github
Only in out/masterfiles/inventory: README.md
Only in out/masterfiles/lib: README.md
Only in out/masterfiles: LICENSE
Only in out/masterfiles/modules: promises
Only in out/masterfiles: .no-distrib
Only in out/masterfiles/services/autorun: README.md
Only in out/masterfiles/templates: README.md
```